iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/XMLExporter.java

```diff
@@ -63,6 +67,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(XMLExporter.class);
 
+    private static final Supplier<ExtensionProviders<ExtensionXmlSerializer>> EXTENSIONS_SUPPLIER = Suppliers.memoize(() -> ExtensionProviders.createProvider(ExtensionXmlSerializer.class, "network"));
```

Comment:
Why do you use memoize here?

Reply:
This supplier provides in runtime the XML serializers of network extensions available in the dependencies of the module in which it is called. `memoize` allows to use lazy loading: the serializers are only loaded when first called (if never called, they are never loaded and once loaded, they are kept in memory).

Reply:
Ok, thank you for the explanation!